.pre-commit-config.yaml

```diff
@@ -58,8 +58,8 @@ repos:
     rev: v1.10.0
     hooks:
       - id: mypy  # See pyproject.toml for args
-        additional_dependencies:
-          - types-all
+        # additional_dependencies:
+        #   - types-all
```

Comment:
Not sure we want to check this in, may relate to #9750 which is an independent issue / workaround